### DISCUSSION TITLE
I do not receive an email address to sign in.

### DISCUSSION THREAD

**Discussion by yukipioa** [mozilla/hubs#5306] [Hubs Cloud Questions]
Hello.
I have a question.

I have already set up the region I am using as an out-of-sandbox environment, but when I enter my email address to sign in, I don't receive an email.
What is the cause and solution?

Many thanks!

**Comment by pattersonbl2** [comment]
Not really a simple reason for this.  You could do a journalctl to see the SMTP side in the server and also check the SES setting in AWS. 

A link to SES troubleshooting [here](https://docs.aws.amazon.com/ses/latest/dg/troubleshoot-error-messages.html)
